src/pages/Build/TermsOfService.vue

```diff
@@ -0,0 +1,188 @@
+				<p>
+					<!-- eslint-disable-next-line max-len -->
+					This Agreement shall continue until terminated as set forth in this Section. Either party may terminate this Agreement at any time, for any reason, or for no reason, including but not limited to, termination by Kiva if Licensee violates any provision of this Agreement or if the exercise of the rights granted hereunder by Licensee is inconsistent with or is in conflict with Kiva’s mission, values, goals or objectives. Any termination of this Agreement shall also terminate the license granted hereunder. Upon termination of this Agreement for any reason, Licensee shall: (a) destroy and remove from all computers, hard drives, networks, and other storage media all copies of the Licensed API, (b) shall so certify to Kiva that such actions have occurred, and (c) cease using and will cease distribution of its Developed Application. Kiva shall have the right to inspect and audit Licensee’s facilities to confirm the foregoing. Sections 3, 7, 9, 10, 11, 13, 14 and 15 shall survive termination of this Agreement.
+				</p>
+				<h2>12. Government Use</h2>
+				<p>
+					<!-- eslint-disable-next-line max-len -->
+					If Licensee is part of an agency, department, or other entity of the United States Government (“Government”), the use, duplication, reproduction, release, modification, disclosure or transfer of the Licensed API is restricted in accordance with the Federal Acquisition Regulations as applied to civilian agencies and the Defense Federal Acquisition Regulation Supplement as applied to military agencies. The Licensed API is a “commercial item,” “commercial computer software” and “commercial computer software documentation,” respectively. In accordance with such provisions, any use of the Licensed API by the Government shall be governed solely by the terms of this Agreement. Manufacturer is Kiva Microfunds.
+				</p>
+				<h2>13. Export Controls</h2>
+				<p>
+					<!-- eslint-disable-next-line max-len -->
+					Licensee shall comply with all relevant export laws and restrictions and regulations of the United States including but not limited to those of the Department of Commerce, the United States Department of Treasury Office of Foreign Assets Control, as well as those of foreign agencies or authorities, and Licensee shall not export, or allow the export or re-export of the Licensed API or any Developed Application in violation of any such restrictions, laws or regulations. Licensee will also not use the Licensed API for any purpose prohibited by such laws, restrictions or regulations (including but not limited to nuclear, chemical or biological weapons proliferation). By using the Licensed API, Licensee agrees to the foregoing and represents and warrants that Licensee is not located in, under the control of, or a national or resident of any restricted country.
+				</p>
+				<h2>14. Governing Law; Resolution of Disputes; Arbitration and Waivers</h2>
+				<p style="font-weight: bold">
```

Comment:
Also here, please remove the inline style.

Wrap the text in a <strong></strong> or <b></b> tag.

Reply:
changed these all to class "legal-bold". and it clears my stylelint locally now. thanks!